src/MongoDB.Driver.Core/Core/Authentication/ScramSha256Authenticator.cs

```diff
@@ -33,6 +33,8 @@ namespace MongoDB.Driver.Core.Authentication
 {
     /// <summary>
     /// A SCRAM-SHA256 SASL authenticator.
+    /// In .NET Standard, this class does not normalize the password in the credentials, so non-ASCII
+    /// passwords may not work unless they are normalized into Unicode Normalization Form KC beforehand.
```

Comment:
Alternatively:

In .NET Standard, this class does not fully SaslPrep the password in the credentials, so non-ASCII passwords may not work. Normalizing the password into Unicode Normalization Form KC beforehand MAY help. SCRAM-SHA-1 is the recommended alternative for now.

Reply:
It's more important to say this in Driver rather than Core.

Also, add it to the release notes.

Reply:
Updated release notes.